src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java

```diff
@@ -43,12 +48,17 @@ public void run(final UserInput userInput) {
         } else if (userInput.getGoal() == Goal.RELEASE) {
             reports.addAll(this.releaseUseCase.release(userInput));
         }
-        // TODO: this should be part of the use cases
+        this.logResults(reports);
         writeResponseToDisk(userInput, reports);
     }
 
+    private void logResults(final List<Report> reports) {
+        for (final Report report : reports) {
+            this.reportLogger.logResults(report);
```

Comment:
It will now log that the validation is successful or not after it runs all releases, won't it? And meanwhile, a user will see some logs about the release process,  so the order is broken:

2020-12-09 16:12:28.691 [INFO   ] Validation started. 
2020-12-09 16:12:32.086 [INFO   ] Release started. 
2020-12-09 16:12:32.917 [INFO   ] Releasing on GITHUB platform. 
2020-12-09 16:12:34.146 [INFO   ] A GitHub workflow 'github_release.yml' has started. The Release Droid is monitoring its progress. This can take from a few minutes to a couple of hours depending on the build. 
2020-12-09 16:13:35.343 [INFO   ] VALIDATION Report: validation is successful! 
2020-12-09 16:13:35.343 [INFO   ] RELEASE Report: release is successful! 
2020-12-09 16:13:35.352 [INFO   ] A full report is available: /.release-droid/last_report.txt

Reply:
right!